src/client/src/consumer.rs

```diff
@@ -37,7 +37,7 @@ use crate::spu::SpuPool;
 ///
```

Comment:
Can you move this as separate PR since this is not related to this purpose of this PR